### DISCUSSION TITLE
Simple Interface to drop in a database schema

### DISCUSSION THREAD

**Discussion by romans127** [directus/directus#6047] [Feature Requests]
Greetings, 

This may already be possible and/or have been vetted but it would be really cool to have a simple monaco editor type interface that allows you to drop in a db schema (with permissions and security in mind) and create very quickly a database schema. 

I see this being helpful for those that simply want to try out a data model really quick in Directus to see how things will flow. Could be a quickstart to a bigger idea or something to build on. 

I imagine there will need to be consideration of security as well as possibly an api endpoint that accepts raw sql then performs a direct database execution against the commands.

**Comment by rijkvanzanten** [comment]
I believe that's more or less what https://github.com/directus/directus/discussions/3891 is discussing :)

**Reply by romans127** [reply]
Awesome thank you
